### DISCUSSION TITLE
Showcasing my Project from my Partner's Repo

### DISCUSSION THREAD

**Discussion by jeffliu27** [community/community#23395] [New to GitHub]
<p>I was jsut wondering what the best way to bring in my project with someone else when the source code is in their GitHub repo. Would it be forking it into my repo? </p>
<p>Or would it just be zipping the project after it’s completed and uploading it into my account? </p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/jeffliu27">@jeffliu27</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our community organized so users can more easily find information.</p>
<p>As you’ll notice, your topic is now here in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/jeffliu27">@jeffliu27</a>,</p>
<p>What specifically do you mean by showcasing this project? Is this a project that you and a partner worked on together which you’re turning in for credit?</p>
<p>You could definitely fork the project which would give you a copy at its present state. </p>
<p>Thanks!</p>

**Comment by jeffliu27** [comment]
<p>Hi <a class="mention" href="/that-pat">@that-pat</a>,</p>
<p>It’s a team project that I make contributions on but it doesn’t show up on my profile as one of my repos. By showcasing I meant if a recruiter asked for my github, they would be able to see the project as one of my projects.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/jeffliu27">@jeffliu27</a>, you can pin a repository to your profile if you contribute to it, even if you don’t own it. To do this, go to your profile and click on  <strong>Customize your pinned repositories</strong>. In the box that pops up, you can select <strong>Show: Repositories that you contribute to</strong>. Then you should be able to select the repository that you worked on as a team. Would this work for your use case?</p>

**Reply by Paojangam** [reply]
so does that make the repositories count go up or no?

**Comment by jeffliu27** [comment]
<p><a class="mention" href="/that-pat">@that-pat</a> Thanks for the input. That’s exactly what I was looking for. </p>
<p>thanks again</p>

**Comment by ThirdEyeRose** [comment]
<p>You’re very welcome, <a class="mention" href="/jeffliu27">@jeffliu27</a>! Glad I could help.</p>

**Comment by saurabh4autonomous** [comment]
<p>Hello <a class="mention" href="/that-pat">@that-pat</a> ,<br>
This is exactly what I am looking for.<br>
Unfortunately, this solution is not working for me.</p>
<p>I am getting <strong>Customize your pins</strong>  instead of <strong>Customize your pinned repositories</strong>.<br>
And on clicking this, the pop up box only shows my repos created by me and forked by me and doesn’t show the Repo I have contributed to.</p>
<p>Please guide me if I am missing anything here.</p>
<p>Thanks &amp; Regards<br>
Saurabh</p>
